src/components/numberinput/Numberinput.vue

```diff
@@ -193,27 +195,27 @@ export default {
         },
         onControlClick(event, inc) {
             // IE 11 -> filter click event
-            if (event.detail !== 0 || event.type === 'click') return
+            if (event.detail !== 0 || event.type !== 'click') return
```

Comment:
You are using `{key, blabla}` rn, so, maybe there's also should be `{detail, type}`?